bin/gaffer

```diff
@@ -122,6 +122,8 @@ prependToPath "$GAFFER_ROOT/glsl" IECOREGL_SHADER_INCLUDE_PATHS
 prependToPath "$GAFFER_ROOT/fonts" IECORE_FONT_PATHS
 prependToPath "$HOME/gaffer/ops:$GAFFER_ROOT/ops" IECORE_OP_PATHS
 
+prependToPath "$GAFFER_ROOT/resources/IECoreUSD" PXR_PLUGINPATH_NAME
```

Comment:
I am not 100% sure but I think this env var might not be a "path" and rather a single file, so might be better to set rather than prepend if that turns out to be the case?

Reply:
From the USD docs :

>Another way USD is locating plugins is the PXR_PLUGINPATH_NAME environment variable. This variable may be a list of paths.`

Otherwise you'd only be able to have one plugin wouldn't you?

Reply:
Oh great, ignore me then.

And yes, I thought they had done it the annoying way where this would be a manifest of all plugins.